### PR TITLE
make some shared package versions acept version range

### PR DESCRIPTION
**Pull request by mkholjuraev**
This is intended to have a wider package version choice while deciding the package version in the shared scope. If the chrome app accepts a wider range of package versions, consumer apps will be freer to choose/update their version. This helps to avoid version mismatches and duplicate packages in the shared scope.

We need to continue with this approach in the future package updates in chrome.

### CODE REVIEW

**Comment by mkholjuraev** [comment]
@Hyperkid123 @karelhala I am not able to ask for your review on GH for some reason that I do now know.

**Comment by Fewwy** [comment]
I have no issues with this PR, but I have a question about 1 thing. Have you forgotten to create a branch for this PR or you created this PR on the master branch for some purpose? :)

**Comment by mkholjuraev** [comment]
@Fewwy yup, this is on purpose. I felt lazy to create a separate branch and delete afterwards...

File: package.json [hunk 1]
```diff
@@ -174,7 +174,7 @@
     "@redhat-cloud-services/frontend-components-utilities": "^3.3.11",
     "@redhat-cloud-services/host-inventory-client": "1.2.0",
     "@redhat-cloud-services/rbac-client": "1.2.0",
-    "@scalprum/core": "0.4.0",
+    "@scalprum/core": "^0.4.0",
```
Comment (Hyperkid123) [inline package.json:177]: This has to be locked as the package is still not stable.

**Review by Hyperkid123 — COMMENTED** [review]
Looks good. Scalprum rules can't be relaxed yet.

File: package.json (hunk 1 above)
Comment (mkholjuraev) [reply package.json]: @Hyperkid123  done!

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**